ansible_collections/arista/avd/roles/eos_designs/doc/network-services.md

```diff
@@ -373,6 +380,11 @@ mac_address_table:
             # Extend this SVI over VXLAN
             vxlan: < true | false | default -> true >
 
+            # Explicitly extend this VLAN to remote EVPN domains.
+            # Overrides tenants.<tenant>.evpn_multi_domain ( default --> true ).
+            # Overrides tenants.<tenant>.vrf.<vrf>.evpn_multi_domain ( default --> true ).
+            evpn_multi_domain: < true | false | default --> true >
```

Comment:
This is not default true. This inherits from vrf, so it has no default

Reply:
Resolved